pages/expect.jsx

```diff
@@ -138,7 +138,7 @@ var Expect = React.createClass({
                   </td>
                   <td>
                     <div className="time">19:30</div>
-                    <div>MozFest Party at the <a href="https://www.thersa.org/contact">RSA</a></div>
+                    <div>MozFest Party</div>
```

Comment:
Changes in this page fixes some of https://github.com/mozilla/mozillafestival.org/issues/821